vignettes/develop_intro.Rmd

```diff
@@ -178,6 +178,11 @@ HTMLWidgets.widget({
           sig.renderers[name].resize(width, height);  
       },
       
+      destroy: function() {
```

Comment:
dispose? only reason I suggest it is if it doesn't matter here, then we use dispose internally so it would match